bin/parse

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env node
-
+const fs = require('fs')
+const functional = require('../lib-functional/index')
```

Comment:
Since you are only using the applyParsingFunctions function, import it like so:
`const applyParsingFunctions = require('../lib-functional/index').applyParsingFunctions`

You can then call applyParsingFunctions directly on line 18.